### PR TITLE
Highcharts integration

### PR DESCRIPTION
**Pull request by birdage**
DO NOT MERGE

### CODE REVIEW

**Comment by birdage** [comment]
@DanielJMaher ill fix the merge conflicts now before i progress anymore

File: ooiui/templates/science/plotting.html [hunk 1]
```diff
@@ -200,7 +217,16 @@
       });
     }
   },
-
+  setDefaults: function() {
```
Comment (birdage) [inline ooiui/templates/science/plotting.html]: @Bobfrat FYI this is how i get the data from the services

**Comment by birdage** [comment]
@FBRTMaka @DanielJMaher @Bobfrat we will want to keep an eye on this as i had to make a few fixes i think i got all the issues

**Comment by birdage** [comment]
@Bobfrat wanna pull this down and give it a go, see how it fits in to what you did

**Comment by Bobfrat** [comment]
surely

**Comment by DanielJMaher** [comment]
@Bobfrat If this is good and you like the code, merge.

**Comment by Bobfrat** [comment]
:+1:

**Pull request merged**